public/mockServiceWorker.js

```diff
@@ -0,0 +1,284 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker.
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
+ */
+
+const PACKAGE_VERSION = '2.3.5'
```

Comment:
This should update as new versions of MSW are installed (remains to be seen if dependency bump PRs would automatically do this, I doubt it).

I also added this to prettierignore, as I had accidentally tweaked the formatting which isn't great